tawazi/_dag/dag.py

```diff
@@ -98,11 +101,31 @@ def __init__(
 
     @property
     def max_concurrency(self) -> int:
-        """Maximal number of threads running in parallel. (will change!)."""
-        return self._max_concurrency
+        """Same as max_threads_concurrency. (DEPRECATED!)."""
+        warnings.warn(
```

Comment:
Having this might be bad for production settings since it'll clog the logs wdyt ? I'd say we formally do a breaking change

Reply:
It will only do the warning once, then It will stop showing the warning